src/widget/wlibrarytableview.cpp

```diff
@@ -121,3 +122,7 @@ void WLibraryTableView::setTrackTableRowHeight(int rowHeight) {
     verticalHeader()->setDefaultSectionSize(math_max(
             rowHeight, fontHeightPx));
 }
+
+void WLibraryTableView::updateGeometries() {
```

Comment:
since librarytableview inherits from qtableview, is this noop override necessary?